samples/bluetooth/broadcast_audio_assistant/prj.conf

```diff
@@ -11,4 +11,8 @@ CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=191
 CONFIG_BT_TINYCRYPT_ECC=y
 
 CONFIG_BT_EXT_ADV=y
+CONFIG_BT_ISO_MAX_CHAN=2
```

Comment:
The broadcast assistant don't need ISO at all - Why are we setting ISO chan to 2?